.github/workflows/publish.yml

```diff
@@ -6,24 +6,30 @@ on:
       - main
 
 jobs:
-  deploy:
```

Comment:
Honestly, I reckon you could probably do exactly the same here as the chromatic job. Leave everything as is and run the entire job _only_ if it's not `chore(release): publish`.

Given they are all reliant on the build steps it just seems easier, that way we don't run applitools either.

Reply:
That makes sense. I've made that change.